views/api.jade

```diff
@@ -153,9 +153,9 @@ block content
                                         if app.hasSubscription && app.subscriptionApproved
                                             td
                                                 if apiInfo.auth == "key-auth"
-                                                    a(href='#{glob.network.schema}://#{glob.network.apiHost}/swagger-ui/?apikey=#{app.apiKey}&url=#{app.swaggerLink}' target='_blank').btn.btn-default Try it!
+                                                    a(href='/swagger-ui/?apikey=#{app.apiKey}&url=#{app.swaggerLink}' target='_blank').btn.btn-default Try it!
```

Comment:
This is still not correct; if the initial thing does not work for you, it will have a different reason. The `/swagger-ui` endpoint is routed via the API Gateway (Kong) so that API calls do not necessarily have to be CORS enabled to work from Swagger UI. But like this, only CORS-enabled APIs will work from Swagger UI.